pkg/kubelet/kubelet.go

```diff
@@ -282,24 +282,21 @@ func makePodSourceConfig(kubeCfg *kubeletconfiginternal.KubeletConfiguration, ku
 }
 
 // PreInitRuntimeService will init runtime service before RunKubelet.
-func PreInitRuntimeService(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
-	kubeDeps *Dependencies,
-	remoteRuntimeEndpoint string,
-	remoteImageEndpoint string) error {
-	// remoteImageEndpoint is same as remoteRuntimeEndpoint if not explicitly specified
-	if remoteRuntimeEndpoint != "" && remoteImageEndpoint == "" {
-		remoteImageEndpoint = remoteRuntimeEndpoint
+func PreInitRuntimeService(kubeCfg *kubeletconfiginternal.KubeletConfiguration, kubeDeps *Dependencies) error {
+	// imageServiceEndpoint is same as containerRuntimeEndpoint if not explicitly specified
+	if kubeCfg.ContainerRuntimeEndpoint != "" && kubeCfg.ImageServiceEndpoint == "" {
+		kubeCfg.ImageServiceEndpoint = kubeCfg.ContainerRuntimeEndpoint
```

Comment:
Will this mutate the kubeCfg? Why not just declare a local var?

Reply:
done